lib/rex/post/sql/ui/console/command_dispatcher/client.rb

```diff
@@ -120,6 +120,10 @@ def cmd_query_help
               def run_query(query)
```

Comment:
At a quick glance it looks like some of this logic is in the wrong place; Does this mean that if the user is in the context of the `query_interactive` command - this logic won't apply? 👀